edit-post/index.js

```diff
@@ -67,6 +68,7 @@ export function initializeEditor( id, post, settings ) {
 	const reboot = reinitializeEditor.bind( null, target, settings );
 
 	registerCoreBlocks();
+	registerCoreInlineBlocks();
```

Comment:
I think we should just move these blocks in the `core-block` folder, even though the API would be slightly different.

Reply:
What about grouping with `RichText`?

Reply:
Actually I was thinking of removing this separate folder and putting it in `core-blocks`, so it would all be registered together. These would have `inline` as the category. How does that sound?

Reply:
Mmm, not sure we should group these with blocks as they are different entities. To me, they are more like utilities for RichText, which is in turn a component.

Reply:
I moved some stuff around. Still polishing.